domain/secret/service/service.go

```diff
@@ -299,7 +315,7 @@ func (s *SecretService) ProcessSecretConsumerLabel(
 	if uri == nil {
 		var err error
 		uri, err = s.GetURIByConsumerLabel(ctx, label, unitName)
-		if errors.Is(err, errors.NotFound) {
+		if errors.Is(err, secreterrors.SecretNotFound) {
 			return nil, nil, errors.NotFoundf("consumer label %q", label)
```

Comment:
errors.NotFoundf("secret with consumer label %q", label)

Reply:
It's not strictly the secret that cannot be found but the consumer record which refers to the secret. This error message is what we've been using till now. I'll see if I can think of a tweak to make to it.

Reply:
reworded it a bit